README.adoc

```diff
@@ -4,13 +4,595 @@
 
 Metanorma plugin that allows you to access static data structures like JSON, YAML, XML from a Metanorma document
 
-=== Installation
+== Installation
 
 [source,console]
 ----
 $ gem install metanorma-plugin-datastruct
 ----
 
-== Documentation
+== Usage
 
-See http://metanorma.com[]
+In order to use macroses(yaml2text, json2text) you need to register them as Asciidoctor extentions in metanorma.
```

Comment:
@w00lf can we write something like:

In order to use the macros in Metanorma, add the gem `gem 'metanorma-plugin-datastruct'` in your Gemfile. Or have the gem installed, and Metanorma can use them automatically via the extension `:plugin: datastruct`.

Reply:
Changed